docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,85 @@
+# <a name="feature-request"></a> Filing a feature request
+A feature request is all you need when you have a spark of inspiration. When filing a feature request you do it in the [issues section](https://github.com/CollActionteam/collaction_app/issues), remember to select the feature request template, or [click here to start filing a feature request](https://github.com/CollActionteam/collaction_app/issues/new?assignees=&labels=&template=feature_request.md&title=%5BSUGGESTION%5D%3A+Your+suggestion+title+here).
+
+Describe your idea in as much detail you can, no need to be super technical, but the better your idea is described, the easier its intent is to understand. Sadly it's not easy to guarantee that all features are as relevant, or all features will be implemented, we have to filter them to select the features with the most value. Don't be discouraged if your first feature request is denied, feel free to try another feature.
+
+Before filing a feature request, try searching in the issues for keywords related to the feature request you have in mind, there is a chance that someone else already filed the same or a similar request!
+# <a name="branches"></a> Branches naming conventions
+We use simple naming conventions for branches, to easily navigate and track multiple branches across features. Branches follow a 3 part logic, the first part describes the type of work is being done on the branch.
+
+<a name="tags"></a>Often used tags are: `feat`, `feature`, `chore`, `task`, `refac`, `refactor`, `fix`, `bugfix`, `docs`, `documentation`, etc.
+
+The second part is what issue or feature the branch relates to. This is used by taking the issue number, and prefixing it with either `gh-` or `can-`.
+
+Examples: `gh-97`, `can-104`, etc.
+
+The third and last part describes the scope, it's okay to be generic if the scope is not easily defined. Multiple branches can have a similar name, and thus the scope helps differentiate and specify what exactly the branch pertains to. All words in the scope must be separated by `-`.
+
+<a name="scopes"></a>Examples: `profile-avatar`, `profile-settings`, `auth-new-user`, `crowdaction-details`, etc.
+
+These three parts are separated by a slash (`/`), and concatenating the parts examples for branch names are:
+`feat/can-104/profile-avatar`, `chore/gh-97/update-readme`, etc.
+
+Don't use underscore (`_`) in branch naming, and don't use uppercase letters.
+# <a name="commits"></a> Commit conventions
+To ensure our commit history stays clean, and can more easily be used for tracking, we follow some simple conventions that go hand in hand with our [branches naming conventions](#branches).
+
+The format for the commit title is:
+ `<tag>(<scope>): <subject>`
+
+The format for a commit with body and footer is:
+```
```

Comment:
Maybe also add a complete example of a commit message (like example branch names above)